pkg/apb/ext_creds.go

```diff
@@ -63,9 +60,10 @@ func monitorOutput(namespace string, podname string, log *logging.Logger) ([]byt
 		}
 
 		log.Warning("[%s] Retry attempt %d: exec into %s failed", podname, r, podname)
-		time.Sleep(time.Duration(credentialExtInterval) * time.Second)
+		time.Sleep(time.Duration(apbWatchInterval) * time.Second)
 	}
-	timeout := fmt.Sprintf("[%s] ExecTimeout: Failed to gather bind credentials after %d retries", podname, credentialExtRetries)
+
+	timeout := fmt.Sprintf("[%s] ExecTimeout: Failed to gather bind credentials after %d retries", podname, apbWatchRetries)
```

Comment:
Renamed because they are now used in two places, one has nothing to do with credentials.